src/vs/vscode.proposed.d.ts

```diff
@@ -2914,4 +2914,102 @@ declare module 'vscode' {
 	export type DetailedCoverage = StatementCoverage | FunctionCoverage;
 
 	//#endregion
+
+
+	//#region https://github.com/microsoft/vscode/issues/128444 --- Type hierarchy --- @eskibear
```

Comment:
Without a main issue, I simply put the link to Iteration Plan for July 2021 here.

Reply:
Let's use this issue: https://github.com/microsoft/vscode/issues/15533